src/main/java/com/iota/iri/conf/TestnetConfig.java

```diff
@@ -53,6 +57,39 @@ protected void setDontValidateTestnetMilestoneSig(boolean dontValidateTestnetMil
         this.dontValidateTestnetMilestoneSig = dontValidateTestnetMilestoneSig;
     }
 
+    @Override
+    public int getNumberOfKeysInMilestone() {
+        return numberOfKeysInMilestone;
+    }
+
+    @JsonProperty("NUMBER_OF_KEYS_IN_A_MILESTONE")
+    @Parameter(names = "--milestone-keys", description = MilestoneConfig.Descriptions.NUMBER_OF_KEYS_IN_A_MILESTONE)
+    protected void setNumberOfKeysInMilestone(int numberOfKeysInMilestone) {
+        this.numberOfKeysInMilestone = numberOfKeysInMilestone;
+    }
+
+    @Override
+    public int getCoordinatorSecurityLevel() {
+        return coordinatorSecurityLevel;
+    }
+
+    @JsonProperty("COORDINATOR_SECURITY_LEVEL")
+    @Parameter(names = "--testnet-coordinator-security-level", description = MilestoneConfig.Descriptions.COORDINATOR_SECURITY_LEVEL)
+    protected void setCoordinatorSecurityLevel(int coordinatorSecurityLevel) {
+        this.coordinatorSecurityLevel = coordinatorSecurityLevel;
+    }
+
+    @Override
+    public SpongeFactory.Mode getCoordinatorSignatureMode() {
+        return coordinatorSignatureMode;
+    }
+
+    @JsonProperty("COORDINATOR_SIGNATURE_MODE")
+    @Parameter(names = "--testnet-coordinator-signature-mode", description = MilestoneConfig.Descriptions.COORDINATOR_SIGNATURE_MODE)
```

Comment:
I originally added a converter and validator, a la compass
https://github.com/iotaledger/compass/blob/3aaf05763cded4a229cc23f8e9cd3d53256948db/compass/conf/BaseConfiguration.java#L49

but the default validation is really good (see PR::test)